dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/collection/CollectionUtils.java

```diff
@@ -57,6 +57,11 @@
 {
     public static final String[] STRING_ARR = new String[0];
 
+    private CollectionUtils()
```

Comment:
Recently, we are given preference to (Lombok annotation)
```
@NoArgsConstructor( access = PRIVATE )
```
instead of declaring the private constructor programatically.

Reply:
Thanks, done!